src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -398,7 +448,9 @@ public static ConstantSet getConstants(String classname, ConstantSet result) {
             case Const.SIPUSH:
               {
                 ConstantPushInstruction cpi = (ConstantPushInstruction) inst;
-                result.ints.add((Integer) cpi.getValue());
+                Integer value = (Integer) cpi.getValue();
```

Comment:
I think you should handle:

dconst_0
dconst_1	
lconst_0
lconst_1
fconst_0
fconst_1
fconst_2
iconst_0
iconst_1
iconst_2
iconst_3
iconst_4
iconst_5

Oh, I see they are above, but not in the diff.

They should be handled by the regular Randoop code too -- I don't agree with Ben's comment that they aren't worth mentioning.  Being common is a very important reason for them to be put in the pool.  We could make that change first, in another pull request, if you want.

Reply:
I have committed https://github.com/randoop/randoop/commit/1b54df5346773f0db9088a640eb431b27535940a to enable you to make this change.
Let me know if you have comments on that pull request.  (I merged it without external review, because I didn't want to slow you down, but review is always helpful.)

Reply:
Thank you for the changes there - I don't have any additional comments for those changes. I've made changes to Constant Mining to also make use of those constant values.